crates/aiken-lang/src/format.rs

```diff
@@ -1799,31 +1799,35 @@ fn printed_comments<'a, 'comments>(
 
     let mut doc = Vec::new();
     while let Some(c) = comments.next() {
-        // There will never be consecutive empty lines (None values),
-        // and whenever we peek a None, we advance past it.
-        let c = c.expect("no consecutive empty lines");
-        doc.push("//".to_doc().append(Document::String(c.to_string())));
-        match comments.peek() {
-            // Next line is a comment
-            Some(Some(_)) => doc.push(line()),
-            // Next line is empty
-            Some(None) => {
-                comments.next();
+        match c {
+            None => continue,
+            Some(c) => {
```

Comment:
The diff is hard to read but I really just added these three lines above and nested the existing logic under the `Some` clause.